extensionsGallery-insider.json

```diff
@@ -2556,6 +2556,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "54",
+					"extensionName": "delete-database",
+					"displayName": "Delete database",
+					"shortDescription": "Adds 'Delete' option when right clicking on a database",
+					"publisher": {
+						"displayName": "AlexP",
+						"publisherId": "alx-ppv",
+						"publisherName": "alx-ppv"
+					},
+					"versions": [
+						{
+							"version": "0.0.3",
+							"lastUpdated": "08/14/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
```

Comment:
Isn't this going to mean users have to go to the page - download it and then install it manually? Is that our SOP for non-first party extensions? It'd be easier to just have this be a VSIX package link and then point at https://github.com/alx-ppv/ADS-delete-database/releases/download/0.0.3/deletedb-0.0.3.vsix if that's not the case then right?

Reply:
Yes, currently 3rd-party extensions need to be manually side-loaded.  Once we have a proper marketplace we plan to improve this experience is so all extensions can be installed directly within the product.